lib/stdlib/doc/src/queue.xml

```diff
@@ -201,12 +248,30 @@
           <c><anno>Item</anno></c> is not copied. If it returns a list,
           the list elements are inserted instead of <c>Item</c> in the
           result queue.</p>
+          <p><em>Example 1:</em></p>
+          <pre>
+1> <input>Queue = queue:from_list([1,2,3,4,5]).</input>
+2> <input>Queue1 = queue:filter(fun (E) -> E > 2, Queue).</input>
+3> <input>queue:to_list(Queue1).</input>
+[3,4,5]</pre>
         <p>So, <c><anno>Fun</anno>(<anno>Item</anno>)</c> returning
           <c>[<anno>Item</anno>]</c> is thereby
           semantically equivalent to returning <c>true</c>, just
           as returning <c>[]</c> is semantically equivalent to
           returning <c>false</c>. But returning a list builds
           more garbage than returning an atom.</p>
+          <p><em>Example 2:</em></p>
+          <pre>
+1> <input>Queue = queue:from_list([1,2,3,4,5]).</input>
+2> <input>Queue1 = queue:filter(fun (E) -> [E, E+1], Queue).</input>
+3> <input>queue:to_list(Queue1).</input>
+[1,2,2,3,3,4,4,5,5,6]</pre>
+          <p><em>Example 3:</em></p>
```

Comment:
Here it makes sense having multiple examples. I am not sure that the third example is really necessary.